src/main/resources/migrations/00001_create_table_payment_requests.sql

```diff
@@ -0,0 +1,16 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-payment_requests
+CREATE TABLE payment_requests (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
To use Big ints and sequence generators that allows preallocation

Reply:
Applies for all entities

Reply:
:+1: will change as part of the next PR